build_cmake/toolchains/aarch64/linux/plat_gcc.cmake

```diff
@@ -0,0 +1,11 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+# See the file LICENSE for redistribution information.
+#
+
+if(CMAKE_CROSSCOMPILING)
+    set(CROSS_COMPILER_PREFIX "aarch64-linux-gnu-" CACHE INTERNAL "" FORCE)
```

Comment:
I'm curious why here it's set as a cache variable, while in the equivalent clang file (plat_clang.cmake) it's set as a normal variable.

Reply:
Great find, we don't need to cache `CROSS_COMPILER_PREFIX`. It doesn't have any effect on the toolchain step. Seems to be method that I copied over to the other gcc toolchain files also. 
I've updated the GCC toolchain files, removing `CACHE INTERNAL "" FORCE`